src/PointwiseFunctions/AnalyticSolutions/Elasticity/HalfSpaceMirror.cpp

```diff
@@ -83,12 +83,12 @@ tuples::TaggedTuple<Tags::Displacement<3>> HalfSpaceMirror::variables(
               },
               lower_boundary, absolute_tolerance_, relative_tolerance_);
       get<2>(result)[i] = displacement_z;
+    } catch (convergence_error& error) {
+      ERROR("The numerical integral in the AnalyticSolution fails for r="
+            << r << ", z=" << z << " (" << error.what()
+            << "). Try increasing IntegrationIntervals or decreasing the "
+               "domain extents.\n");
```

Comment:
```suggestion
            << "). Try to increase the 'IntegrationIntervals' or make the domain smaller.\n");
```

(The word "extents" often refers to the number of grid points in other places in the code, so it can be ambiguous)